pkg/fileutil/util_test.go

```diff
@@ -122,3 +122,34 @@ func TestDeleteDynamicFile(t *testing.T) {
 	}
 	testA.mutex.Unlock()
 }
+
+func TestCalculateTimeDeltaFromUnixInSeconds(t *testing.T) {
+	type args struct {
+		from string
+		to   string
+	}
+	cases := []struct {
+		input args
+		want  float64
+	}{
+		{
+			args{"1706648530", "1706648539"},
+			9.0,
+		},
+		{
+			args{"1706648520", "1706648539"},
+			19.0,
+		},
+	}
+
+	for _, c := range cases {
```

Comment:
can you add a negative test case that invalid format?

Reply:
added